addons/stock/product.py

```diff
@@ -203,9 +203,15 @@ def _product_available_text(self, cr, uid, ids, field_names=None, arg=False, con
 
     def _compute_nbr_reordering_rules(self, cr, uid, ids, field_names=None, arg=None, context=None):
         res = {}
-        product_data = self.pool['stock.warehouse.orderpoint'].read_group(cr, uid, [('product_id', 'in', ids)], ['product_id'], ['product_id'], context=context)
+        product_data = self.pool['stock.warehouse.orderpoint'].read_group(cr, uid, [('product_id', 'in', ids)], ['product_id', 'product_min_qty', 'product_max_qty'], ['product_id'], context=context)
         for data in product_data:
-            res[data['product_id'][0]] = int(data['product_id_count'])
+            res[data['product_id'][0]] = {'nbr_reordering_rules': int(data['product_id_count'])}
+            if int(data['product_id_count']) == 1:
+                res[data['product_id'][0]].update({
+                    'reordering_min_qty' : data['product_min_qty'],
+                    'reordering_max_qty' : data['product_max_qty']
+                })
+
```

Comment:
As this is a computed field, I think you have to return the reordering_min_qty and reordering_max_qty even if product_id_count > 1. These field values make sense even if product_id_count > 1, it's just that you are going to use product_id_count to discriminate between one display or another in the view, but here ? You don't care, just return the fields informations.